src/cc_catalog_airflow/dags/europeana_workflow.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from provider_api_scripts import europeana
+from util.operator_util import get_log_operator
+
+
+DAG_DEFAULT_ARGS = {
+    'owner': 'data-eng-admin',
+    'depends_on_past': False,
+    'start_date': datetime(1970, 1, 1),
+    'email_on_retry': False,
+    'retries': 3,
+    'retry_delay': timedelta(minutes=15),
+}
+
+DAG_ID = 'europeana_workflow'
+
+
+def get_runner_operator(dag):
+    return PythonOperator(
+        task_id='pull_europeana_data',
+        python_callable=europeana.main,
+        op_args=['{{ ds }}'],
+        depends_on_past=False,
+        dag=dag
+    )
+
+
+def create_dag():
+    dag = DAG(
+        dag_id=DAG_ID,
+        default_args=DAG_DEFAULT_ARGS,
+        start_date=datetime(1970, 1, 1),
+        schedule_interval='@monthly',
```

Comment:
This should be daily, no?
```suggestion
        schedule_interval='@daily',
```

Reply:
I believe we settled on fetching the database monthly as per our conversation on Slack and [this comment](https://github.com/creativecommons/cccatalog/issues/279#issuecomment-611077266).

Reply:
@mathemancer  I concur with your preference. The ingestion method seems to be the most appropriate in our case. 
However, I'm unable to work on this at the moment, so you could take it up.
Thank you!